components/NivoBarChart.tsx

```diff
@@ -0,0 +1,46 @@
+import { ResponsiveBar } from '@nivo/bar';
+
+interface NivoBarChartProps {
+  name: string;
+  items: Array<
+    {
+      itemName: string;
+    } & any
```

Comment:
what is the point of this? shouldn't it just be any?

Reply:
It was an issue with the fact that one attribute name can only determined at runtime. Should be addressed in the next commit.